client/src/boot/registerComponents.js

```diff
@@ -100,5 +101,6 @@ export default () => {
     NumberField,
     PopoverOptionSet,
     ToastsContainer,
+    CmsSiteName
```

Comment:
This bits register our demo component in injector. Any new custom component would have to go through this step.

I nice thing here is that a developer could choose to override our component with their own custom implementation.